src/components/PeopleTable.tsx

```diff
@@ -3,642 +1,85 @@
+import { Person } from '../types';
+import { PersonLink } from '../components/PersonLink';
+import { SearchLink } from './SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+
+  const sortType = searchParams.get('sort');
+  const sortOrder = searchParams.get('order');
+
+  const findParentPerson = (parentName: string) => {
+    return people.find(person => {
+      return person.name === parentName;
+    });
+  };
+
+  const getSortParam = (sortParam: string) => {
+    if (sortType === sortParam && sortOrder) {
+      return { sort: null, order: null };
+    }
+
+    if (sortType === sortParam) {
+      return { sort: sortParam, order: 'desc' };
+    }
+
+    return { sort: sortParam, order: null };
+  };
```

Comment:
The `getSortParam` function is a bit confusing. It's not clear why you return `null` for both `sort` and `order` when the current sort type matches the passed in sort param and `sortOrder` is truthy. A comment explaining the logic here would be helpful.